ui/app/AppLayouts/Chat/controls/community/HoldingTypes.qml

```diff
@@ -4,4 +4,12 @@ QtObject {
     enum Type {
         Asset, Collectible, Ens
     }
+
+    enum Mode {
+        Add, Update
+    }
+
+    enum FlowType {
```

Comment:
This enum seems to be used only internally by `HoldingsDropdown` and imo should be defined there.

Reply:
Ok, agree!